Mathlib/Geometry/Manifold/IntegralCurve.lean

```diff
@@ -259,6 +314,23 @@ lemma isIntegralCurveOn_comp_mul_ne_zero {a : ℝ} (ha : a ≠ 0) :
   · simp only [smul_smul, inv_mul_eq_div, div_self ha, one_smul]
   · simp only [mem_setOf_eq, mul_assoc, inv_mul_eq_div, div_self ha, mul_one, setOf_mem_eq]
 
+-- TODO: missing lemma(s) about Ioo
```

Comment:
As above: add the lemmas or excise the comment if it turns out there aren't any appropriate lemmas.